__tests__/app.test.js

```diff
@@ -78,6 +78,41 @@ describe("GET: /api/articles/:article_id", () => {
           expect(article).toHaveProperty("article_img_url");
         });
     });
+    test("Should return article 3 with the total comment count on it", () => {
```

Comment:
We don't need to be this specific in our tests, we are testing general behaviours not the specifics of our test database. So here, I'd say you need one test : GET 200 responds with comment_count property